plexapi/base.py

```diff
@@ -40,23 +56,49 @@ class PlexObject:
             initpath (str): Relative path requested when retrieving specified `data` (optional).
             parent (:class:`~plexapi.base.PlexObject`): The parent object that this object is built from (optional).
     """
-    TAG = None      # xml element tag
-    TYPE = None     # xml element type
-    key = None      # plex relative url
+    TAG: Optional[str] = None  # xml element tag
+    TYPE: Optional[str] = None  # xml element type
+    key: Optional[str] = None  # plex relative url
+    _INCLUDES = {
```

Comment:
Moving `_INCLUDES` here causes them to be added to all `PlexObject` in `_buildDetailsKey`. The includes should only be added to the details key for `PlexPartialObjects`.

Reply:
that is something i did not know. fixed by including an empty member on `PlexObject` to satisfy pyright